package.json

```diff
@@ -18,7 +18,7 @@
   "bugs": "https://github.com/xjamundx/eslint-plugin-promise/issues",
   "scripts": {
     "format": "prettier --write .",
-    "lint": "eslint .",
+    "lint": "eslint . && prettier -c .",
```

Comment:
I don't know if I agree with this one change. what do you see as the benefit here? Is this similar to `eslint --fix` ?

Reply:
ESLint only checks the JS files, but prettier is run on everying. The lint-staged will catch stuff if people have made the commit from a setup local environment, but not from the web. This way the full formatting check gets run in CI